gulpfile.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var gulp = require('gulp');
+var jeditor = require('gulp-json-editor');
+var merge = require('merge-stream');
+var zip = require('gulp-zip');
+
+var files = ['background.js', 'contentScript.js', 'logo.png',
```

Comment:
I'm not comfortable making that kind of large-scale architectural change to this repo without @notwaldorf's okay.
(I'll also note that making that change would break the check-out-and-run workflow that currently exists.  I'm not sure how important that is, and don't feel qualified to decide one way or the other…  :wink:)